spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/paths/BenchmarkPaths.java

```diff
@@ -53,20 +53,28 @@ public String parquetDir(int scale) {
         return String.join(File.separator, rootDataDir(scale), "raw_parquet");
     }
 
-    public String experimentResultLocation(int scale, String queryName) {
-        return String.join(File.separator, rootDataDir(scale), "experiments", StringUtils.removeEnd(queryName, ".sql"));
+    public String experimentResultLocation(String experimentName, int scale, String queryName) {
```

Comment:
`experimentName` gets plumbed through a lot of places, but in the end just always gets passed as an argument to `BenchmarkPaths`, which itself has no fields (so is effectively a utility class of methods that could be static). 

Feels like either `experimentName` should be passed to the constructor of `BenchmarkPaths` so that we don't have to plumb it through all the places where we want to compute a path, or BenchmarkPaths should become a proper utility class (so we don't have to plumb a `paths` object through all the classes). Either way, passing both things around all the places seems silly.